spec/models/grantee_spec.rb

```diff
@@ -26,25 +26,49 @@
 
     describe "#name" do
       it "delegates to the attributes" do
-        expect(subject.name).to eq hses_grantee[:attributes][:name]
+        expect(subject.name).to eq hses_grantee[:attributes][:granteeName]
       end
     end
 
     describe "#region" do
       it "delegates to the attributes" do
-        expect(subject.region).to eq hses_grantee[:attributes][:region]
+        expect(subject.region).to eq hses_grantee[:attributes][:grants][0][:attributes][:region]
```

Comment:
[commiseration] dang this grantees > grants thing really made things more complicated :/